tests/ApplicationTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Console\Tests;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Console\Event\ApplicationStartup;
+use Yiisoft\Yii\Console\Event\ApplicationShutdown;
+
+final class ApplicationTest extends TestCase
+{
+    public function testGetDispatcher(): void
+    {
+        $this->assertInstanceOf(EventDispatcherInterface::class, $this->application->getDispatcher());
+    }
+
+    public function testDispatcher(): void
```

Comment:
I'm not sure this test is needed. In fact, it tests event dispatcher, not console. And the even dispatcher already has its own tests.

Reply:
done.